ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -908,5 +908,26 @@
     "CaseEventID": "allocatedJudge",
     "UserRole": "caseworker-publiclaw-cafcass",
     "CRUD": "R"
+  },
+  {
```

Comment:
Immediately live or toggle off at first?

Reply:
I see no reason why it can't go live immediately